# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: parser/htmlparser/tests/mochitest/mochitest.ini
# Commit: 9a8abd87cc79
# Full Hash: 9a8abd87cc7935f29b94248c1a6f8203faa14403
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-08 21:43:06
# Regressor Bug: 1701828
# File Overlap Count: 1
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/parser/htmlparser/tests/mochitest/mochitest.ini b/parser/htmlparser/tests/mochitest/mochitest.ini
--- a/parser/htmlparser/tests/mochitest/mochitest.ini
+++ b/parser/htmlparser/tests/mochitest/mochitest.ini
@@ -21,16 +21,25 @@ support-files =
   file_bug672453_late_meta.html
   file_bug672453_meta_non_superset.html
   file_bug672453_meta_restart.html
   file_bug672453_meta_unsupported.html
   file_bug672453_meta_utf16.html
   file_bug688580.js
   file_bug672453_not_declared.html
   file_bug672453_meta_userdefined.html
+  file_bug672453_xml_decl.html
+  file_bug672453_meta_after_head.html
+  file_bug672453_meta_replacement.html
+  file_bug672453_http_replacement.html
+  file_bug672453_http_replacement.html^headers^
+  file_bug672453_enc_error.html
+  file_bug672453_enc_error_inherited.html
+  file_bug672453_meta_speculation_fail.html
+  file_bug672453_xml_speculation_fail.html
   file_bug716579-16.html
   file_bug716579-16.html^headers^
   file_bug716579-16.xhtml
   file_bug716579-16.xhtml^headers^
   file_bug716579-8.html
   file_bug716579-8.html^headers^
   file_bug716579-8.xhtml
   file_bug716579-8.xhtml^headers^